ksql-engine/src/main/java/io/confluent/ksql/codegen/CodeGenRunner.java

```diff
@@ -123,35 +121,39 @@ public ExpressionMetadata buildCodeGenFromParseTree(
       this.functionRegistry = functionRegistry;
     }
 
+    private void addParameter(Optional<Field> schemaField) {
+      parameters.add(new ParameterType(
+          SchemaUtil.getJavaType(schemaField.get().schema()),
+          schemaField.get().name().replace(".", "_")));
```

Comment:
Consider encapsulating this `. -> _` translation into a semantically named --and thus self explanaing-- method inside `SchemaField`. This would help code readability. As it stands, why do you need to do this `replace` operation?

Reply:
schemaField is an instance of Field which is a Connect class so I won't be able to change it from KSQL side.